src/update_agent/actor.rs

```diff
@@ -94,6 +95,19 @@ impl UpdateAgent {
         ctx.notify_later(RefreshTick {}, after)
     }
 
+    /// Apply a small, random amount (0% to 10%) of jitter to a given period.
```

Comment:
This is creating and returning the duration for jittering, not actually applying it, correct?

Reply:
This is doing the sum internally and returning the total, but not mutating the fixed period in input.
I've slightly renamed this, it should be a bit clearer now.